VictoriaAlbert/VictoriaAlbert/VictoriaAlbertViewController.swift

```diff
@@ -0,0 +1,90 @@
+
+
+class VictoriaAlbertTableViewController: UITableViewController {
+    var museum: Museum?
+    var museumFields = [Museum]()
+    var detailImgOfCell = String()
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        loadMusem(apiEndpoint: endpoint)
+        
+    }
+    
+    func loadMusem(apiEndpoint: String) {
+        APIRequestManager.manager.getData(endPoint: apiEndpoint) { (data: Data?) in
+            if let validData = data,
+                let validMuseum = Museum.buildMuseum(from: validData) {
+                self.museumFields = validMuseum
+                DispatchQueue.main.async {
+                    self.tableView.reloadData()
+                }
+            }
+        }
+    }
+    
+    
+    //MARK: - TableView funcs
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return museumFields.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "VaCell", for: indexPath)
+        if let vaCell = cell as? VATableViewCell {
+            let selectedCell = museumFields[indexPath.row]
+            
+            vaCell.vaLabel?.text = "\(selectedCell.object), \(selectedCell.date), \(selectedCell.place)"
+            
+            
+            //GETTING IMAGE DATA ************** 2007BM/2007BM5536_jpg_o.jpg"
+            let imageEndpoint = "http://media.vam.ac.uk/media/thira/collection_images/\(selectedCell.imageIdShort)/\(selectedCell.imageId)_jpg_o.jpg"
+            
+            APIRequestManager.manager.getData(endPoint: imageEndpoint, callback: { (data: Data?) in
+                guard let unwrappedImageData = data else { return }
+                DispatchQueue.main.async {
+                    vaCell.vaImage?.image = UIImage(data: unwrappedImageData)
+                    vaCell.setNeedsLayout()
+                }
+            })
+            
+            
+        }
+        return cell
+    }
+    
+    //Segue
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let selectedCell = museumFields[indexPath.row]
+        detailImgOfCell = "http://media.vam.ac.uk/media/thira/collection_images/\(selectedCell.imageIdShort)/\(selectedCell.imageId).jpg"
+        self.performSegue(withIdentifier: "vaSegue", sender: selectedCell)
+    }
+    
+    //MARK: - Segue to DetailViewController
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "vaSegue" {
+            if let vc = segue.destination as? DetailViewController {
+                vc.detailMuseumField = museumFields
+                vc.detailImgEndpoint = detailImgOfCell
+            }
```

Comment:
So, what you need to do to use a segue alone is to cast the sender as a cell and get its index path from the tableView. This way you can access the appropriate index in your data array. Also, I'd prefer you passed the whole Museum object to the detail view. This is more flexible for future additions.